src/plugins.js

```diff
@@ -0,0 +1,131 @@
+export default {
+  // es2015
+  "transform-es2015-arrow-functions": {
+    edge: 13,
+    firefox: 45,
+    chrome: 49,
```

Comment:
Function.name won't work correctly, at least in some cases, with untransformed arrow functions. `const x = () => {}; x.name` may become `"(anonymous)"`, but it should be `"x"`.